app/models/queen.rb

```diff
@@ -1,13 +1,13 @@
 # Queen behavior.
 class Queen < Piece
-  # Capture, collision, check, and checkmate logic are not
+  # Capture, check, and checkmate logic are not
   # implemented yet and thus ignored.
   def valid_move?(x, y)
     moved?(x, y) &&
       (
```

Comment:
I'm not sure how to go about this since I don't have a collection, just 3 method calls.  Is there a way to implement a collection of method calls that would make sense here to reduce complexity?